src/power_consumption/model.py

```
@@ -1,6 +1,9 @@
 from sklearn.ensemble import RandomForestRegressor
 from sklearn.pipeline import Pipeline
 from sklearn.metrics import mean_squared_error, r2_score
+import mlflow
+import pandas as pd
+from power_consumption.utils import adjust_predictions
```

Ruff

`power_consumption.utils.adjust_predictions` imported but unused

Remove unused import: `power_consumption.utils.adjust_predictions`

(F401)